egs/librispeech/asr/simple_v1/mmi_att_transformer_train.py

```diff
@@ -75,7 +78,7 @@ def get_objf(batch: Dict,
 
     grad_context = nullcontext if is_training else torch.no_grad
 
-    with grad_context():
+    with autocast(enabled=scaler.is_enabled()), grad_context():
```

Comment:
at validation step, the scaler is None. scaler.is_enabled() will fail because 'NoneType' object has no attribute 'is_enabled'

Reply:
Sorry and thanks! I made it work on the cluster and then didn't test after adding the disable switch and merging conflicts. It's fixed now in https://github.com/k2-fsa/snowfall/pull/175